src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -974,34 +837,23 @@ public IStorageMonitorable getInventory( IActionSource src )
 
 	private class FluidHandler implements IFluidHandler
 	{
+		private final IFluidTankProperties[] TANK_PROPS = new IFluidTankProperties[] { new FluidTankProperties( null, Fluid.BUCKET_VOLUME ) };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "TANK_PROPS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00116)